actions/redoc/template.hbs

```diff
@@ -66,14 +98,16 @@
     window.dataLayer = window.dataLayer || [];
     function gtag(){dataLayer.push(arguments);}
     gtag('js', new Date());
-
     gtag('config', 'UA-41294851-1');
   </script>
+
+  <script src="js/determine_widget.js"></script>
   <script>
-    var ffWidgetId = '16a23fe0-c29a-48ec-a531-5123fd9f60bf';
+    var ffWidgetId = getId(window.location.host);
+    console.log("Yeetaroni!");
     var ffWidgetScript  = document.createElement('script');
     ffWidgetScript.type = 'text/javascript';
-    ffWidgetScript.src = 'https://cdn.freddyfeedback.com/widget/freddyfeedback.js';
```

Comment:
could i get more details on this cdn change? I'm just curious about what cdn is and wondering if it's ok to change the link given in freddyfeedback's installation tutorials

Reply:
That is the link in the installation instructions: https://freddyfeedback.com/dashboard/survey/16a23fe0-c29a-48ec-a531-5123fd9f60bf?date_range=last_30_days